.github/workflows/upload-deb-images.yml

```diff
@@ -0,0 +1,56 @@
+name: Upload package creation Docker images - DEB - amd64 and i386
+on:
+  pull_request:
+    paths:
+      - 'debs/Debian/**'
+      - 'debs/build.sh'
+  push:
+    branches:
+      - master
+      - 4.5
+      - 4.4
+    paths:
+      - 'debs/Debian/*'
+      - 'debs/build.sh'
+  workflow_dispatch:
+
+jobs:
+  Upload-deb-package-building-images:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [ {CONTAINER_NAME: deb_builder_amd64, DOCKERFILE_PATH: debs/Debian/amd64}, {CONTAINER_NAME: deb_builder_i386, DOCKERFILE_PATH: debs/Debian/i386}]
```

Comment:
If the i386 docker file is modified, does the workflow rework the x86_64 image?

Reply:
Changed in https://github.com/wazuh/wazuh-packages/pull/2053/commits/ad78e4b494e07a7b48c61a454968f82e1fa0b745. Now if only one Dockerfile has been changed, only that image will be uploaded